posts/2021-01-22-jwtbuilder-redirects-21001.adoc

```diff
@@ -0,0 +1,182 @@
+
+MicroProfile Config 2.0 is a major version release (from 1.4) which introduces Functional and API/SPI changes in Microprofile Config 2.0.
+
+For a more in-depth look at the new capabilities offered by MicroProfile 2.0, take a look at the full explanation in the 20.0.0.12-beta blog post, link:https://openliberty.io/blog/2020/10/27/microprofile40-jakartaee9-beta-200012.html#conf[here].
+
+
+To enable MicroProfile Config 2.0 in your app, add the following to your `server.xml`:
+
+[source, xml]
+----
+<featureManager>
+   <feature>mpConfig-2.0</feature>
+</featureManager>
+----
+
+[#bugs]
+== Notable bugs fixed in this release
+
+We’ve spent some time fixing link:https://github.com/OpenLiberty/open-liberty/issues?q=label%3Arelease%3A200011+label%3A%22release+bug%22[bugs in 21.0.0.1], including the following issues:
+
+* link:https://github.com/OpenLiberty/open-liberty/issues/14936[Improved recovery when core components are reinstalled at runtime]
++
+Previously, Open Liberty contained a flaw in the detection of changes to the JARs that compose the server implementation. When such changes were detected Liberty would force the JAR to be uninstalled and installed again. In most cases this allowed the runtime to recover and function properly. In specific cases, where some core component was re-installed, Liberty would not properly recover and it would result in some Java packages to be unavailable for class loading.  For example, on Java 11 the package `javax.xml.soap` would become unavailable to the application class loaders. 
++
+The most common environment where this occurred was running Open Liberty in OpenShift. This behavior has now been corrected for Open Liberty 21.0.0.1.
+
+* link:https://github.com/OpenLiberty/open-liberty/issues/14936[Enable MyFaces 2.3.7 for Open Liberty]
++
+The `jsf-2.3` feature in Open Liberty makes use of link:https://myfaces.apache.org/core23/index.html[Apache MyFaces] JavaServer Faces implementation. With the release of Apache MyFaces 2.3.7 a number of improvements and bug fixes have been made, for more information visit the link:https://issues.apache.org/jira/secure/ReleaseNote.jspa?projectId=10600&version=12346524[Apache MyFaces 2.3.7 changelog].
+
+* link:https://github.com/OpenLiberty/open-liberty/issues/15337[Dynacache initialization issue when ID is missing]
++
+An external user discovered that the initialization of a `distributedMap` fails with a NullPointerException if the `id` element is not present in the `distributedMap` configuration.
+A clearer message should have been displayed to indicate the required `id` element is missing.  To address this the `distributedMap` definition has been modified to mark the `id` element as required.
++
+The configuration runtime will now issue an error message: 
++
+`CWWKG0058E: The element distributedMap with the unique identifier default-0 is missing the required attribute id.`
```

Comment:
Might be better in a codeblock but double check with someone first

Reply:
Agreed, changed.